packages/docs/content/slush-wallet/dapp.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: dApp Integration
+---
+
+<Callout type="info">
+	If you are using `dapp-kit`, you do not need to install any additional packages to integrate with
+	the Slush wallet. [Read how to integrate with dapp-kit.](../dapp-kit/slush)
+</Callout>
+
+Using the Slush Wallet SDK, you can allow users to connect to the Slush wallet from your dApp. The
+wallet is provided through the [Wallet Standard](https://docs.sui.io/standards/wallet-standard), and
+should appear automatically in your existing wallet connection UI.
+
+## Considerations
+
+- Users will only be able to connect their zkLogin wallet from Slush.
+- The zkLogin account will be managed by Slush, you do not need to set up any OAuth providers.
+- You will not get access to the user's private key, you will only be able to connect to the wallet,
+  and suggest transactions and messages to sign, just like any other wallet in the ecosystem.
+
+## Setup
+
+To use the Slush wallet, you will need to register it in your application, using
+`registerSlushWallet`. This only needs to be done once, and should be done as early as possible in
+your application's lifecycle.
+
+`registerSlushWallet` takes two arguments:
+
+- `name`: The name of your dApp. This will be shown to the user when they are asked to approve the
+  connection in Slush.
+- `options`: An optional object with the following properties:
+  - `origin`: The origin of the Slush website. Defaults to `https://getslush.com`.
```

Comment:
Let's remove any URLs that we don't own from here just to avoid someone thinking this is the domain and scammers buying it.